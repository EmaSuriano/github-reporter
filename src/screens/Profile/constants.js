const DEFAULT_OPTIONS = {
  maintainAspectRatio: false
};

export const CHART_CONFIGURATION = {
  height: 250,
  width: 450,
  options: DEFAULT_OPTIONS
};

export const MIN_LANGUAGES = 1;
