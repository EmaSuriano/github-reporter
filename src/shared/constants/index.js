const DEFAULT_ERROR = "No results found";

export { DEFAULT_ERROR };
