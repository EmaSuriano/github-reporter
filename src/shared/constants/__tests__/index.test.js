import constants from "../";

describe("Constants", () => {
  it("should be match with the structure of the snapshot", () =>
    expect(constants).toMatchSnapshot());
});
