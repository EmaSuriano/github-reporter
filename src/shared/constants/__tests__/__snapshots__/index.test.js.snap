// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants should be match with the structure of the snapshot 1`] = `undefined`;
